app/src/main/java/com/keylesspalace/tusky/TuskyApplication.kt

```diff
@@ -69,10 +70,10 @@ class TuskyApplication : Application(), HasAndroidInjector {
         AppInjector.init(this)
 
         // Migrate shared preference keys and defaults from version to version. The last
-        // version that did not have a SCHEMA_VERSION was 97, so that's the default.
-        val oldVersion = sharedPreferences.getInt(PrefKeys.SCHEMA_VERSION, 97)
-        if (oldVersion != BuildConfig.VERSION_CODE) {
-            upgradeSharedPreferences(oldVersion, BuildConfig.VERSION_CODE)
+        // version that did not have a SCHEMA_VERSION was 100, so that's the default.
+        val oldVersion = sharedPreferences.getInt(PrefKeys.SCHEMA_VERSION, 100)
+        if (oldVersion != SCHEMA_VERSION) {
+            upgradeSharedPreferences(oldVersion, SCHEMA_VERSION)
```

Comment:
I would probably call it "SETTINGS_VERSION" or similar (schema is more headed for apis and databases).

Reply:
This is a schema though -- it defines the allowed keys, the data types of their values, and how those values are to be interpreted.

Implementation details (are they stored in a KV store, protobufs, an SQL database...?) are orthogonal to the fact that underlying data absolutely has a schema that governs how the data should be interpreted.

Reply:
It's already in `SettingsConstants.kt`, and its use is limited to this file. I'm not sure this warrants an additional namespace.